Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,284 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
+        private const string InValidOutput = "N/A.";
+
+        public static bool IsInValidValidInput(string[] input) => input == null || input.Length == 0;
+
         /// <summary>
         /// Return name and balance(current) of person who had the biggest historic balance.
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (IsInValidValidInput(peopleAndBalances))
+            {
+                return InValidOutput;
+            }
+
+            var personWithHighestBalance = new StringBuilder();
+            var highestBalanceEver = FindHighestBalance(peopleAndBalances, personWithHighestBalance);
+
+            if (personWithHighestBalance.ToString().Contains(", "))
+            {
+               ReplaceCommaWithAnd(personWithHighestBalance); 
+            }
+
+            return $"{personWithHighestBalance.ToString()} had the most money ever. ¤{highestBalanceEver}.";
+        }
+
+        private static void ReplaceCommaWithAnd(StringBuilder message)
+        {
+            var index = message.ToString().LastIndexOf(", ");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4967975)

Reply:
I can do that but then I break tests with more then 1 person.  I tried to add CurrentCulture and InvariantCulture but both break tests

Reply:
and I makes no sense to use here a locale.  Here it is string,  string, string and that is independ on the localae.

Reply:
Strings also differ sometimes.

Reply:
oke, so you mean I have to use a locale ?